theano/gpuarray/corr_gemm.c

```diff
@@ -191,6 +191,165 @@ KERNEL void col2im_kernel(const ga_size n,
   }
 }
 
+#kernel dilated_im2col_int_kernel : int, *, int, int, int, int, int, int, int, int, int, int, int, int, int, * :
```

Comment:
no, I don't want regression compared to the old back-end. The old back-end was using int, not uint. I can use ga_int, no problem with that. I recall some special case of uint on the GPU.